e2e/scenario_test.go

```diff
@@ -1201,39 +1126,3 @@ func Test_ubuntu1804imdsrestriction_mangletable(t *testing.T) {
 		},
 	})
 }
-
-func Test_Ubuntu2204MessageOfTheDay(t *testing.T) {
```

Comment:
will wanna check with @UtheMan on these, I believe they were recently added

Reply:
That should be fine, added it as separate test since its a separate feature, but if we are fine with combining these cases like this then i'm all for it.

Reply:
We should have a Happy Path test where nothing gets configured and we ensure that none of those settings are getting set. 

We could look at combining multiple features into another single e2e test to minimize the number of tests

Reply:
While it looks like we didn't set nothing, it's far from truth: 

https://github.com/Azure/AgentBaker/blob/master/e2e/template.go#L13-L449